tests/bluetooth/controller/ctrl_periodic_sync/prj.conf

```diff
@@ -0,0 +1,50 @@
+CONFIG_ZTEST=y
+
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+CONFIG_ASSERT_VERBOSE=y
+
+CONFIG_BT=y
+CONFIG_BT_HCI=y
+CONFIG_BT_CTLR=y
+CONFIG_BT_CTLR_PRIVACY=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_LLL_VENDOR_NORDIC=y
+
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CTLR_PERIPHERAL_ISO=y
+CONFIG_BT_CTLR_CENTRAL_ISO=y
+
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_ACL_RX_SIZE=251
+CONFIG_BT_DATA_LEN_UPDATE=y
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_BT_ASSERT=y
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+CONFIG_BT_CTLR_PER_INIT_FEAT_XCHG=y
+CONFIG_BT_CTLR_SCA_UPDATE=y
+CONFIG_BT_SCA_UPDATE=y
+
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT=n
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_SYNC_TRANSFER_SENDER=y
+CONFIG_BT_CTLR_SYNC_TRANSFER_RECEIVER=y
+CONFIG_BT_CTLR_SYNC_ISO=y
+CONFIG_BT_CTLR_SCAN_SYNC_ISO_SET=1
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT=1
+CONFIG_BT_CTLR_CONN_ISO_STREAMS=1
+CONFIG_BT_CTLR_CONN_ISO_GROUPS=1
```

Comment:
Do we need ISO for this test?

Reply:
Yes, as the `sca` field of the `struct lll_sync` is flaged with `#if defined(CONFIG_BT_CTLR_SYNC_ISO)` and is needed for PAST for work

Reply:
Hmm, that sounds odd. ISO and PAST are unrelated features, and there shouldn't be a dependency there. Maybe the `sca` field need to be available for non-ISO builds? @cvinayak

Reply:
@lutb-ot you should make `sca`  available for PAST support too. 

Background, periodic sync needs `sca`  but uses it as local variable to calculate window widening. But a ISO sync is established using an inherited `sca` value, hence `sca` was stored in `lll_sync` context. Now that PAST feature will need the `sca` to be remembered, i think we should make the `sca` field available in the `lll_sync` context?  or wait, if this is for window widening, then it would be calculated only once, do we really need it to be present in `lll_sync` context?

Reply:
Yeah, so `sca` should not depend on ISO, and thus should be updated any ISO requirements from these tests should be removed

Reply:
I'll make `sca` depend on PAST too and update/fix ISO requirements in the bsim and unit tests.

Reply:
Yes, I sadly didn't make it before my vacation as there where more dependencies that I thought. I have fixed it now.